ansible/configs/ocp-workloads/ocp-workload.yml

```diff
@@ -17,7 +17,17 @@
     set_fact:
       ansible_python_interpreter: "/opt/virtualenvs/k8s/bin/python"
 
-  - name: Run workload
+  - name: Run single workload
     include_role:
       name: "{{ ocp_workload }}"
     when: ocp_workload is defined
+
+  - name: Run multiple workloads
+    when:
+      - ocp_workloads is defined
+      - ocp_workloads | length > 0
+    loop: "{{ ocp_workloads }}"
+    loop_control:
+      loop_var: _ocp_workload
+    include_role:
+      name: "{{ _ocp_workload }}"
```

Comment:
@wkulhanek Do you see an issue with having something like this in here? It allows us to test overriding `ocp_workloads` as well as a single `ocp_workload`. Which is useful as for local dev we want to run Minio before the main `integreatly` role

Reply:
@aidenkeating That's a good idea. Had written up a whole thing.... until I realized that the variable is different( workloads vs workload) (plural vs singular). I think this is good.